protocol/communities/community.go

```diff
@@ -1209,15 +1221,33 @@ func canDeleteMessageForEveryonePermissions() map[protobuf.CommunityMember_Roles
 	return roles
 }
 
-func (o *Community) GetMemberAdmins() []*ecdsa.PublicKey {
-	admins := make([]*ecdsa.PublicKey, 0)
+func (o *Community) GetPrivilegedMembers() []*ecdsa.PublicKey {
+	return append(o.GetOwnerMembers(), append(o.GetTokenMasterMembers(), o.GetAdminMembers()...)...)
+}
+
+func (o *Community) GetAdminMembers() []*ecdsa.PublicKey {
+	return o.getMembersByPredicate(o.IsMemberAdmin)
+}
+
+func (o *Community) GetTokenMasterMembers() []*ecdsa.PublicKey {
+	return o.getMembersByPredicate(o.IsMemberTokenMaster)
+}
+
+func (o *Community) GetOwnerMembers() []*ecdsa.PublicKey {
+	return o.getMembersByPredicate(o.IsMemberOwner)
```

Comment:
do we need to filter out the owner, who is the control node in order not to send 2 requests to join?